pkg/resources/create.go

```diff
@@ -100,8 +100,12 @@ func Create(logger *zap.SugaredLogger, rt json.RawMessage, triggerName, eventID,
 		Resource: apiResource.Name,
 	}
 
-	_, err = dc.Resource(gvr).Namespace(namespace).Create(data, metav1.CreateOptions{})
-	return err
+	logger.Infof("Creating resource with payload: %v", data)
```

Comment:
I wonder if should be logging the whole payload here...just looking at the integration test logs, the payload can he huge (especially if we are passing through large chunks of the incoming body)....maybe we log the gvk and the eventID by default...and if the resource creation fails, then we log the whole payload? (Another option would be to log at the Debug level instead of Info)

Reply:
ah yeah good call - ill make it shorter for now. as a follow up, i think it could be useful to make these payloads available on disk somewhere for debugging (i hacked that into the github interceptor to get this payload)